pages/common/streamlit.md

```diff
@@ -0,0 +1,20 @@
+# streamlit
+
+> Framework for creating interactive, data-driven web apps in Python.
+> More information: <https://docs.streamlit.io/>.
+
+- Check Streamlit installation:
+
+`streamlit hello`
+
+- Run your Streamlit application:
+
+`streamlit run {{project_name}}`
+
+- See all supported Streamlit commands:
+
+`streamlit --help`
+
+- Check the current version of Streamlit:
```

Comment:
```suggestion
- Display version:
```
We generally use the same description for help and version command across all pages.

Reply:
Sure!